source/extensions/filters/http/transformation/body_header_transformer.cc

```diff
@@ -24,17 +24,16 @@ void BodyHeaderTransformer::transform(
     json_body["body"] = body.toString();
   }
 
-  json &headers = json_body["headers"];
-  header_map.iterate(
-      [&headers](const Http::HeaderEntry &header) -> Http::HeaderMap::Iterate {
-        headers[std::string(header.key().getStringView())] =
-            std::string(header.value().getStringView());
```

Comment:
see how the old code made new strings in the headers map? i think we should do the same when parsing headers and multi value headers

Reply:
Done here: https://github.com/solo-io/envoy-gloo/pull/207/commits/922b07899c3bdbe007ef8e0692a1a02f42b6255d